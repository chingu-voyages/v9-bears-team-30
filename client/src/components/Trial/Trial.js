import React from "react"

const Trial = ({text}) => {
    return (
        <div>
            <h1 data-testid="heading-text">This is a Test</h1>
            <p data-testid="inject-text">{text}</p>
        </div>
    )
}

export default Trial

