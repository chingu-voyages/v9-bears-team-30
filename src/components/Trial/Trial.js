import React from "react"

const Trial = () => {
    return (
        <div>
            <h1 data-testid="heading-text">This is a Test</h1>
        </div>
    )
}

export default Trial

