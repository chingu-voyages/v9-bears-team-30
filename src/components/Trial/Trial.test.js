import React from "react"
import { render, getByTestId } from "@testing-library/react"
import "jest-dom/extend-expect"

import Trial from "./Trial"

describe("Trial", () => {
    it("renders", () => {
        const { asFragment } = render(<Trial />)
        expect(asFragment()).toMatchSnapshot()
    })

    it("should render with proper heading", () => {
        const { container } = render(<Trial />)
        const headingText = getByTestId(container, "heading-text")
        expect(headingText.innerHTML).toEqual("This is a Test")
    })
})