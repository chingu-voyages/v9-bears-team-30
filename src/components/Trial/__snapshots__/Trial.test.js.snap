// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trial renders 1`] = `
<DocumentFragment>
  <div>
    <h1
      data-testid="heading-text"
    >
      This is a Test
    </h1>
  </div>
</DocumentFragment>
`;
